File: csharp/02_variables.cs
using System;

class Program {
    static void Main() {
        string name = "ulas";
        int age = 20;
        double height = 1.75;
        bool isStudent = true;

        Console.WriteLine($"{name} {age} {height} {isStudent}");

        age = 21;
        Console.WriteLine("new age: " + age);
    }
}

File: rust/02_variables.rs
fn main() {
    let name = "ulas";          // &str
    let mut age = 20;           // mut so we can change
    let height = 1.75;
    let is_student = true;

    println!("{} {} {} {}", name, age, height, is_student);

    age = 21;
    println!("new age: {}", age);
}
